Codeinwp/neve
# Various Issues

### Summary
Title & meta spacing adjustments;
Combine title and meta in single block under ordering components;
Fix media query for customizer responsive settings;
New comments appearance;


### Will affect visual aspect of the product

YES

### Test instructions


- Test the ordering blocks - ideally backwards compatibility testing with an older version of the theme. For me it worked fine but I really think somebody else should test it just to be sure;
- Check if media query based settings are applied properly now (e.g. container width on tablet);
- Check the spacing of meta and title on Archive and Single Post;
- Check out the comments new style (feedback appreciated cc: @rodica-andronache);


Closes #153.
Partly closing #134(but not entirely).
